### ISSUE TITLE
Slack rich message formatting

### ISSUE THREAD

**Issue by scottcunningham** [chrisgillis/slackwolf#34]
Hey there,

First of all: thanks a lot for implementing this! This has been a lot of fun for my team.

I wanted to make a small feature suggestion: using [Slack's rich message formatting](https://api.slack.com/docs/formatting). Using it could make the messages from the bot a bit easier to read. [Here](https://api.slack.com/docs/formatting/builder?msg=%7B%22attachments%22%3A%5B%7B%22title%22%3A%22%3Amemo%3A%20Town%20Ballot%22%2C%22pretext%22%3A%22%22%2C%22text%22%3A%22%5Cn-%20%3Aknife%3A%20Kill%20%40username1%5Cn-%20%3Aknife%3A%20Kill%20%40username2%5Cn%5Cn%5Cn%3Ahourglass%3A%20*Remaining%20voters*%3A%20None%5Cn%3Anewspaper%3A%20With%20pitchforks%20in%20hand%2C%20the%20townsfolk%20killed%20%40username3%20(Werewolf)%22%2C%22mrkdwn_in%22%3A%5B%22text%22%2C%22pretext%22%5D%7D%5D%7D) is a short demo using Slack's message formatting builder of how you could use this for the "Town Ballot" announcement (just an example, of course). I think it looks pretty good:

![Example](http://i.imgur.com/1K5Wbq6.png)

I'm totally unfamiliar with PHP so I'd have no idea where to start implementing this, so thought I'd see how you felt about it.

**Comment by chrisgillis** [comment]
I agree this kind of output would be a lot more clean. May need to do some refactoring to have all of the data ready to output at one time.
